chevah/keycert/ssh.py

```diff
@@ -299,7 +299,7 @@ def fromString(cls, data, type=None, passphrase=None):
             # Most probably some parts are missing from the key, so
             # we consider it too short.
             raise BadKeyError('Key is too short.')
-        except (struct.error,  binascii.Error, TypeError):
+        except (struct.error,  binascii.Error, TypeError, AssertionError):
```

Comment:
why are we catching an assertion error here ... together with all other errors ?

if assertionerror should be handled, the it should be raised as something else.